### DISCUSSION TITLE
Zero-length entrances/exits, for train platforms/custom station layouts.

### DISCUSSION THREAD

**Discussion by coastercrab** [OpenRCT2/OpenRCT2#18689] [Feature requests]
Basically, letting guests enter the station directly instead of through a building/blank square.

Rotating an entrance kind of delivers this, but it resets every time you enter the build menu, and it often breaks (on the newest build as of posting, when I attempt to rotate an exit it just disappears and a blank error tooltip pops up).

Not sure if this would be practical, just having issues making a multi-platform station with an island platform right now because I need to use more space than I have lol

**Comment by cheweytoo** [comment]
I agree that this could be useful. It can already be done, albeit in a rather hackish way, see [merged_entrance.zip](https://github.com/OpenRCT2/OpenRCT2/files/10107262/merged_entrance.zip).

- build ride with normal entrances and exits
- use tile inspector's "copy and paste" to clone the entrance/exit ONTO the station track
- use tile inspector to delete original entrance/exit
- in tile inspector, click "make usable" in cloned entrance's/exit's properties

You can turn on another entrance style than "no entrance" in the park to see where the entrance actually is now.

The guests' slightly awkward crossing of the train when boarding or unboarding is unavoidable though. And it will likewise not survive further ride construction, so you should do this last after finishing the track.
